### ISSUE TITLE
M_SITE_LOGO: image should have alternative text

### ISSUE THREAD

**Issue by lpirl** [mosra/m.css#135]
`base.html` generates an `img` tag without an `alt` attribute. However, the w3c requires images to have alternative texts. If we don't want to introduce a new configuration option, we could at least set the alternative text to "logo of `SITENAME`" or similar.

**Comment by mosra** [comment]
Good idea. I searched around a bit ([1](https://stackoverflow.com/a/2153684). [2](https://www.drupal.org/project/panopoly/issues/2905972)) and decided on providing an empty alt (`alt=""`). Standards or validity aside, from the accessibility PoV the logo is non-essential, so saying "logo of SITENAME" is just noise, especially when SITENAME is repeated right after. From what I understood, omitting the alt altogether conveys the meaning of "we didn't bother with accessibility", so an empty alt suggests that "we care about accessibility but this image is not important, ignore it".

Pushed as 76f5921f35f35db62574da16d019dce4e9b7ca41, updating all three themes.
